validator/testdata/feature_tests/empty.out

```diff
@@ -19,6 +19,6 @@ feature_tests/empty.html:1:0 The mandatory tag 'noscript > style[amp-boilerplate
 >> ^~~~~~~~~
 feature_tests/empty.html:1:0 The mandatory tag 'body' is missing or incorrect. (see https://amp.dev/documentation/guides-and-tutorials/learn/spec/amphtml#required-markup)
 >> ^~~~~~~~~
-feature_tests/empty.html:1:0 The mandatory tag 'amphtml engine v0.js script' is missing or incorrect. (see https://amp.dev/documentation/guides-and-tutorials/learn/spec/amphtml#required-markup)
->> ^~~~~~~~~
 feature_tests/empty.html:1:0 The mandatory tag 'noscript enclosure for boilerplate' is missing or incorrect. (see https://amp.dev/documentation/guides-and-tutorials/learn/spec/amp-boilerplate?format=websites)
+>> ^~~~~~~~~
```

Comment:
I'm not 100% clear on why these changed position, but my guess is that `mandatory` gets higher precedence in sorting than `mandatory_alternatives`